Automattic/wp-calypso
# Adding Hotjar tracking to the Woo Express specific plans page

Related to #

## Proposed Changes

* This will initialize Hotjar tracking on `/plans/woo-test-site.wpcomstaging.com`
* It uses the existing `loadTrackingTool`, which calls `addHotJarScript`. This function is already aware of user consent (GDPR/opt-out), so it's safe to call like this.

## Testing Instructions



* Via the browser console, run `localStorage.debug='calypso:analytics:hotjar' (for seeing debug messages in the console.
* Go to `/plans/woo-test-site.wpcomstaging.com` where `woo-test-site` is the subdomain of a site with Woo Express trial. If you don't have one, go ahead and create one.
* When the page has loaded, observe that you see a message like `calypso:analytics:hotjar Loading HotJar script +0ms` in the console, and you can also check that window.hj is defined.
* Make sure ad-tracking is enabled

## Pre-merge Checklist
